api/src/main/java/org/openmrs/module/registrationcore/RegistrationCoreUtil.java

```diff
@@ -45,6 +52,23 @@ else if (days == null || days == 0){
 
         return c.getTime();
     }
-
-
+    
+    /**
+     * Get the month from which the birth dates are estimated, defaults to January
+     *
+     * @return
+     */
+    public static int getEstimationStartMonth() {
+        int startMonth = Calendar.JANUARY;
+        String gpStartMonth = Context.getAdministrationService().getGlobalProperty(RegistrationCoreConstants.GP_BIRTHDATE_ESTIMATION_START_MONTH);
+        if (StringUtils.isNotBlank(gpStartMonth)) {
+            try {
+                startMonth = Integer.parseInt(gpStartMonth);
+            } catch (NumberFormatException nfe) {
+                log.warn("Unable to parse value of " + RegistrationCoreConstants.GP_BIRTHDATE_ESTIMATION_START_MONTH + " " + nfe);
```

Comment:
I thought that instead of concatenating nfe, you would pass it as the second parameter in log.warn(message, Throwable)